migrations/capcons/linkmigration.go

```diff
@@ -187,6 +194,15 @@ func (m *LinkValueMigration) Migrate(
 	), nil
 }
 
+func storageKeyToPathValue(
+	storageKey interpreter.StorageKey,
+	storageMapKey interpreter.StorageMapKey,
+) interpreter.PathValue {
+	domain := common.PathDomainFromIdentifier(storageKey.Key)
+	stringStorageMapKey, _ := storageMapKey.(interpreter.StringStorageMapKey)
```

Comment:
is the emtpy string a valid key? shouldn't we panic here if the conversion fail?

Reply:
The idea here was that we _try_ to convert the storage key and storage map key to a path, if it is one (valid domain, and storage map key must be a string). `PathDomainFromIdentifier` will return `PathDomainUnknown` if the storage key is not a path domain, and ignoring the error result in the path identifier being empty. Effectively, this is exactly the same as the "Go default value" for `PathValue` (all fields, domain and identifier, are empty/have the default value).

I'll make it clearer and return an additional bool

Reply:
Tried to improve it in 478e9e0ebec1256672483b2b2a35a9b0e10a67d7